backend/redis_client.py

```
@@ -261,6 +274,9 @@ async def get_cluster_votes(
 
             logger.debug(f"Retrieved {len(votes)} votes from cluster")
             return votes
+        except asyncio.TimeoutError:
+            logger.error(f"Get cluster votes timeout ({self.timeout}s exceeded)")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)